jasmine-2.5.2/spec/BonusSpec.js

```diff
@@ -0,0 +1,30 @@
+describe("Bonus", function() {
+  var bonus;
+
+  beforeEach(function() {
+    bonus = new Bonus();
+  });
+  it('initialises default settings', function() {
+    expect(bonus.getStrikeOrSpare()).toEqual("");
+    expect(bonus.getNextMultiplier()).toEqual(0);
+    expect(bonus.getNextButOneMultiplier()).toEqual(0);
+  });
+  it('records strikes', function() {
+    bonus.recordStrikeOrSpare(1);
+    expect(bonus.getStrikeOrSpare()).toEqual("Strike");
+    expect(bonus.getNextMultiplier()).toEqual(1);
+    expect(bonus.getNextButOneMultiplier()).toEqual(1);
+  });
+  it('records spares', function() {
+    bonus.recordStrikeOrSpare(2);
```

Comment:
I'm not convinced by the use of these magic numbers `1` and `2` to record strikes/spares. I wonder if you could do this more descriptively somehow.